class.jetpack.php

```diff
@@ -4072,13 +4072,30 @@ public function deactivate_dialog( $hook ) {
 			if ( count( $active_plugins_using_connection ) > 1 ) {
 
 				add_thickbox();
+
+				wp_register_script(
+					'jp-tracks',
+					'//stats.wp.com/w.js',
+					array(),
+					gmdate( 'YW' ),
+					true
+				);
+
+				wp_register_script(
+					'jp-tracks-functions',
+					plugins_url( '_inc/lib/tracks/tracks-callables.js', JETPACK__PLUGIN_FILE ),
+					array( 'jp-tracks' ),
+					JETPACK__VERSION,
+					false
+				);
```

Comment:
Could we take that opportunity to register those scripts only once here:
https://github.com/Automattic/jetpack/blob/d25f37a67b0074df3be96536c963601dae89438b/class.jetpack.php#L1157-L1164

And then only add `jp-tracks-functions` as dependency here like you've done, and here:
https://github.com/Automattic/jetpack/blob/3010b47431d784ba4771188ca1da2637bb922d97/class.jetpack-idc.php#L266-L320

Reply:
Yes!

I was planning exactly that but thought I should do it in another PR and make the other places where these scripts are enqueued use this approach